include/openthread/joiner.h

```diff
@@ -106,8 +106,10 @@ typedef void (*otJoinerCallback)(otError aError, void *aContext);
  * @param[in]  aCallback         A pointer to a function that is called when the join operation completes.
  * @param[in]  aContext          A pointer to application-specific context.
  *
- * @retval OT_ERROR_NONE              Successfully started the Commissioner role.
+ * @retval OT_ERROR_NONE              Successfully started the Joiner role.
+ * @retval OT_ERROR_BUSY              The previous attempt is still on-going.
  * @retval OT_ERROR_INVALID_ARGS      @p aPskd or @p aProvisioningUrl is invalid.
+ * @retval OT_ERROR_INVALID_STATE     The IPv6 stack is not enabled or Thread stack is fully enabled.
```

Comment:
```suggestion
 * @retval OT_ERROR_INVALID_STATE     The IPv6 interface is not enabled or Thread stack is fully enabled.
```
I think Thread stack is the IPv6 stack. `ifconfig up` only brings up the IPv6 interface.

Reply:
No, ifconfig up brings up the IPv6 stack.